tests/cli/workflow/balsamic/conftest.py

```diff
@@ -398,23 +396,23 @@ def balsamic_store(base_store: Store, balsamic_dir: Path, helpers) -> Store:
         _store,
         internal_id="balsamic_case_tgs_paired",
         family_id="balsamic_case_tgs_paired",
+        data_analysis="BALSAMIC",
     )
     sample_case_tgs_paired_tumor = helpers.add_sample(
         _store,
         internal_id="sample_case_tgs_paired_tumor",
         is_tumour=True,
         application_tag="TGSA",
         application_type="tgs",
-        data_analysis="BALSAMIC",
         reads=10,
+        data_analysis="MIP+BALSAMIC",
```

Comment:
This should not be possible right?

Reply:
I don't think that is within the scope of this PR, do you?

Reply:
I thought that it was already removed so it should not be tested for, or am I wrong @henrikstranneheim ?

Reply:
This can be set to BALSAMIC only

Reply:
There should be no "MIP+BALSAMIC" fields on case level, each case has a single analysis. To quote Highlander: "There can only be one". If we want to run 2 different analyses - then there should be 2 cases.